scalr/resource_scalr_webhook.go

```diff
@@ -0,0 +1,232 @@
+package scalr
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	scalr "github.com/scalr/go-scalr"
+)
+
+func resourceScalrWebhook() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceScalrWebhookCreate,
+		Read:   resourceScalrWebhookRead,
+		Update: resourceScalrWebhookUpdate,
+		Delete: resourceScalrWebhookDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"enabled": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+
+			"last_triggered_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"events": {
+				Type:     schema.TypeList,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Required: true,
+			},
+
+			"endpoint_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"workspace_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"environment_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func getResourceScope(scalrClient *scalr.Client, workspaceID string, environmentID string) (*scalr.Workspace, *scalr.Environment, *scalr.Account, error) {
```

Comment:
Three return values plus error is a bit too much. Can we create a new private struct with workspace, environment and account and return it instead?